flow/Makefile

```diff
@@ -718,6 +732,7 @@ $(eval $(call do-copy,5_route,4_cts.sdc,,.sdc))
 
 .PHONY: do-route
 do-route:
+	mkdir -p $(RESULTS_DIR) $(LOG_DIR) $(REPORTS_DIR)
```

Comment:
How could you get to these later steps without even generating these directories?  The input files are coming from $(RESULTS_DIR) so how could it not exist?

Reply:
A bit overeager w copy and paste. I can remove the results dir.

reports and logs are trickier. If I use Bazel, then I only have files in the results folder when e.g. I get the synthesis artifact.

Reply:
ok

Reply:
Do you want me to fix it? I will probably, over the next month, find some more nits to fix...

Reply:
I suppose it doesn't hurt and can wait if you wish.